lab_1_classify_by_unigrams/main.py

```diff
@@ -12,13 +12,28 @@ def tokenize(text: str) -> list[str] | None:
     :return: a list of lower-cased tokens without punctuation
     """
 
+    if not isinstance(text, str):
+        return None
+
+    split_text = list(text.lower())
+    tokens = [i for i in split_text if i.isalpha()]
```

Comment:
you can return without creating the variable

Reply:
fixed